core/src/main/java/org/elasticsearch/common/io/stream/StreamOutput.java

```diff
@@ -503,6 +503,18 @@ public void writeOptionalStreamable(@Nullable Streamable streamable) throws IOEx
         }
     }
 
+    /**
+     * Serializes a potential null value.
+     */
+    public void writeOptionalWritable(@Nullable Writeable writeable) throws IOException {
```

Comment:
Same question as the symmetric read [method](https://github.com/elastic/elasticsearch/pull/15111/files#r47225057).